graviti/dataframe/frame.py

```diff
@@ -476,3 +470,28 @@ def sample(self, n: Optional[int] = None, axis: Optional[int] = None) -> "DataFr
 
     def info(self) -> None:
         """Print a concise summary of a DataFrame."""
+
+    def extend(self, objs: "DataFrame") -> None:
```

Comment:
should `extend` supports the pylist input?

Reply:
that means DataFrame.extend add columns, and DataFrame.loc.extend add rows?

Reply:
DataFrame columns more like a dict, and dict extend should be `update`, so I think `DataFrame.extend` is not a good name.